### PR TITLE
Update zh_CN translation

### PR DESCRIPTION
**Pull request by arisac**

### CODE REVIEW

**Comment by arisac** [comment]
@Gudahtt  I believe this is a merge-ready version

**Comment by Gudahtt** [comment]
@arisac: Thanks for the contribution! I'll find somebody to review this as soon as the conflicts are resolved.

We recently added JSON sorting to our formatter, so now there are many conflicts. Would you mind rebasing this onto `develop` and running `yarn && yarn lint:fix`? That should fix everything.

**Comment by arisac** [comment]
> @arisac: Thanks for the contribution! I'll find somebody to review this as soon as the conflicts are resolved.
> 
> We recently added JSON sorting to our formatter, so now there are many conflicts. Would you mind rebasing this onto `develop` and running `yarn && yarn lint:fix`? That should fix everything.

@Gudahtt thanks for notify.I've done rebase and solved conflicts. 
I noticed @dependabot[bot] is failing the CLA test. It should be added to CLA Test whitelist?

**Comment by Gudahtt** [comment]
Thanks again @arisac but something went wrong in your attempt to rebase. You've now duplicated ~56 commits from `develop`, and there are far more changes than there should be.

Could you check the "Allow edits from maintainers" checkbox on this PR? (see here: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork ). That would give me permission to fix things up myself.

**Comment by Gudahtt** [comment]
@arisac Thanks for granting access! I've just rebased your initial 5 commits onto `develop`. It looks correct to me - let me know if anything seems off.

**Comment by Gudahtt** [comment]
I noticed that 3 messages were unused, so I removed them in https://github.com/MetaMask/metamask-extension/pull/9388/commits/c2153dce0750e473ab77840759a317149a4f01e5

**Comment by Gudahtt** [comment]
Ok that was my last update :sweat_smile: I had taken a closer look at your changes, and realized you had made a bunch of changes in that last attempt to update this branch. They were discarded when I rebased just the first 5 commits. I've just restored them all just as you had them.

**Comment by arisac** [comment]
Thank you and sorry for the trouble, that was quite a lot work.

**Review by miranda76 — DISMISSED** [review]

**Comment by arisac** [comment]
I'm currently adding new translation strings to this PR. will update soon.

**Comment by arisac** [comment]
I've updated this PR to sync with the latest develop branch. 
Still more improvements to be done to be a nicely localized in zh_cn translation, but I think this is a good start for the community to contribute on.

**Comment by rekmarks** [comment]
Hi @arisac, thank you for creating this monumental PR. Do you think you'd be willing to rebase it one more time? We recently received some more resources for translation-related work, and can finally review this properly.

很对不起我们花了这么长时间。🙇🏼‍♂️

**Comment by arisac** [comment]
@rekmarks done with new strings added.

I saw [ci/circleci: test-e2e-firefox](https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/7190/workflows/1ee72a0c-38f1-485f-b19b-7b0d9594b6a5/jobs/236732) is getting a failed result due to a timeout.

**Comment by DDDDDanica** [comment]
@rekmarks @arisac  Is it possible to have a comparison file with translated Chinese sentence and original English one?
I can see some potential improvements but it is very difficult which is better without the context.
Thanks ~

**Comment by rekmarks** [comment]
@DDDDDanica, I (or someone else at MetaMask) can make you something by tomorrow!

@arisac, if you created any comparison files while making this, please do let us know.

**Comment by arisac** [comment]
@DDDDDanica @rekmarks I've uploaded the diff file to  https://github.com/arisira/metamask-extension/blob/diff/en-zh_CN.diff using `app/_locales/en/messages.json app/_locales/zh_CN/messages.json >> en-zh_CN.diff`

I removed description key from my translation file since it is not needed for translation, by removing the key, it also helps to diff the translation with the English file because it will pop up in the diff.

And also I've noticed #9772 changed the `gas` back to `gas`. To my opinion, it should be translated, and I'm using `燃料` instead of `天然气` since cn.etherscan.io is using `燃料` as well and I believe it's a good one.

File: app/_locales/zh_CN/messages.json [hunk 1]
```diff
@@ -24,13 +30,19 @@
     "message": "账户选项"
   },
   "accountSelectionRequired": {
-    "message": "您需要选择一个账户 ！"
+    "message": "需要选择一个账户 ！"
```
[the remark this answers is not in the document]
Comment (arisac) [reply app/_locales/zh_CN/messages.json]: Thank you for the guideline. I've updated `你` with `您` in all strings on e86b5dabbe6b3816fb4370490cde598b9d3704ac

File: app/_locales/zh_CN/messages.json [hunk 2]
```diff
@@ -63,45 +75,85 @@
...
+    "message": "已批准"
   },
   "asset": {
     "message": "资产"
   },
+  "assets": {
+    "message": "资产"
+  },
   "attemptToCancel": {
-    "message": "想要取消吗？"
```
[the remark this answers is not in the document]
Comment (arisac) [reply app/_locales/zh_CN/messages.json:89]: Updated in e86b5dabbe6b3816fb4370490cde598b9d3704ac

File: app/_locales/zh_CN/messages.json [hunk 3]
```diff
@@ -63,45 +75,85 @@
...
+    "message": "尝试取消？"
   },
   "attemptToCancelDescription": {
-    "message": "确认提交该操作无法保证能够成功取消您的原始交易。如果取消成功，您将被收取上述交易费用。"
+    "message": "确认提交该操作无法保证能够成功取消你的原始交易。如取消成功，你将被收取上述交易费。"
```
[the remark this answers is not in the document]
Comment (arisac) [reply app/_locales/zh_CN/messages.json]: Updated in e86b5dabbe6b3816fb4370490cde598b9d3704ac

File: app/_locales/zh_CN/messages.json [hunk 4]
```diff
@@ -3,13 +3,19 @@
...
+  },
+  "acceptTermsOfUse": {
+    "message": "我已阅读并同意 $1"
+  },
+  "accessAndSpendNotice": {
+    "message": "$1 可以访问并使用此最大数额"
   },
   "accessingYourCamera": {
-    "message": "正在获取相机使用权限..."
+    "message": "正在访问相机..."
```
[the remark this answers is not in the document]
Comment (arisac) [reply app/_locales/zh_CN/messages.json]: Updated in e86b5dabbe6b3816fb4370490cde598b9d3704ac

File: app/_locales/zh_CN/messages.json [hunk 5]
```diff
@@ -657,52 +1014,88 @@
...
+    "message": "“$1”会关闭此标签，直接回到 $2"
+  },
+  "onlyAddTrustedNetworks": {
+    "message": "恶意的 Ethereum 以太坊网络提供商可以伪造区块链状态，并记录你的网络活动。只添加你信任的自定义网络。"
+  },
+  "onlyAvailableOnMainnet": {
+    "message": "仅在主网（mainnet）上提供"
+  },
+  "onlyConnectTrust": {
+    "message": "只连接你信任的网站。"
```
[the remark this answers is not in the document]
Comment (arisac) [reply app/_locales/zh_CN/messages.json]: Updated in e86b5dabbe6b3816fb4370490cde598b9d3704ac

File: app/_locales/zh_CN/messages.json [hunk 6]
```diff
@@ -657,52 +1014,88 @@
...
+    "message": "确定"
   },
   "on": {
     "message": "启用"
   },
+  "onboardingReturnNotice": {
+    "message": "“$1”会关闭此标签，直接回到 $2"
+  },
+  "onlyAddTrustedNetworks": {
+    "message": "恶意的 Ethereum 以太坊网络提供商可以伪造区块链状态，并记录你的网络活动。只添加你信任的自定义网络。"
```
[the remark this answers is not in the document]
Comment (arisac) [reply app/_locales/zh_CN/messages.json]: Updated in e86b5dabbe6b3816fb4370490cde598b9d3704ac

File: app/_locales/zh_CN/messages.json [hunk 7]
```diff
@@ -1009,165 +1469,488 @@
...
+    "message": "如果价格在你下单和确认之间发生变化，这就叫做“滑点”。如果滑点超过你的“最大滑点”设置，你的的兑换将自动取消。"
+  },
+  "swapAggregator": {
+    "message": "聚合商"
+  },
+  "swapAmountReceived": {
+    "message": "保证数额"
+  },
+  "swapAmountReceivedInfo": {
+    "message": "这是你将收到的最低数额。你也可能会收到更多，这取决于滑点。"
```
[the remark this answers is not in the document]
Comment (arisac) [reply app/_locales/zh_CN/messages.json]: Updated in e86b5dabbe6b3816fb4370490cde598b9d3704ac

File: app/_locales/zh_CN/messages.json [hunk 8]
```diff
@@ -1009,165 +1469,488 @@
...
+  },
+  "swapApproval": {
+    "message": "批准 $1 的兑换 "
+  },
+  "swapApproveNeedMoreTokens": {
+    "message": "你还需 $1 $2 来完成这笔兑换"
+  },
+  "swapBetterQuoteAvailable": {
+    "message": "有一个更好的报价可用"
```
[the remark this answers is not in the document]
Comment (arisac) [reply app/_locales/zh_CN/messages.json]: Updated in e86b5dabbe6b3816fb4370490cde598b9d3704ac

File: app/_locales/zh_CN/messages.json [hunk 9]
```diff
@@ -1009,165 +1469,488 @@
...
+    "message": "“$1”是我们预计的实际产生费用。具体数额视网络情况而定。"
+  },
+  "swapEstimatedNetworkFees": {
+    "message": "预计网络手续费"
+  },
+  "swapEstimatedNetworkFeesInfo": {
+    "message": "这是预估的用于完成你的兑换所使用的网络手续费。实际数额可能会根据网络条件而变化。"
+  },
+  "swapFailedErrorDescription": {
+    "message": "你的资金是安全的，仍然可以在你的钱包中使用。"
```
[the remark this answers is not in the document]
Comment (arisac) [reply app/_locales/zh_CN/messages.json]: Updated in e86b5dabbe6b3816fb4370490cde598b9d3704ac

File: app/_locales/zh_CN/messages.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (arisac) [reply app/_locales/zh_CN/messages.json]: All `...` is updated with the Chinese punctuation `……` in e86b5dabbe6b3816fb4370490cde598b9d3704ac

**Comment by arisac** [comment]
rebased with the latested develop branch with 73bc553ecf99369a7dc09026fb87817bf5e2c2e1 since one string was removed from the English file: `yourSigRequested`

**Comment by arisac** [comment]
@DDDDDanica request for review.

File: app/_locales/zh_CN/messages.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (DDDDDanica) [reply app/_locales/zh_CN/messages.json]: 💯

**Review by DDDDDanica — APPROVED** [review]
All looks good to me 👍

**Review by rekmarks — APPROVED** [review]
With @DDDDDanica's approval, I'm going to merge this.

Thank you so much for taking the time to do this @arisac, and thank you @DDDDDanica for reviewing!

**Pull request merged**